package.json

```diff
@@ -71,6 +73,7 @@
     "abi-decoder": "^1.2.0",
     "abortcontroller-polyfill": "^1.3.0",
     "await-semaphore": "^0.1.1",
+    "babel-loader": "^8.0.6",
```

Comment:
Does this need to be here vs `devDependencies`?

Reply:
```sh
ERROR in multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true
Module not found: Error: Can't resolve 'babel-loader' in '/home/ricky/projects/metamask-extension'
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true main[2]
```

good question! in my notes I had originally meant to double check, but ya looks like it's required.

Reply:
Are you sure? Even the docs for `babel-loader` recommend installing it as a dev dependency. I can't imagine why it'd be needed in the actual application bundle, given that it's a webpack plugin.

Reply:
Oh, sorry I misread! it can certainly be moved over